### ISSUE TITLE
iflytek.py 任务跑不起来

### ISSUE THREAD

**Issue by Lufffya** [bojone/CLUE-bert4keras#4]
我的环境：
tensorflow-gpu == 2.4.0
并使用 tf.keras

我的操作：
修改bert地址，下载数据集，运行 iflytek.py

得到结果：
![image](https://user-images.githubusercontent.com/49063302/148065411-f4e18abd-e9c3-46c7-af2a-2986672234dc.png)

我的尝试：
把 model.compile(loss='sparse_categorical_crossentropy', optimizer=optimizer, metrics=['accuracy'])
里面的optimizer  AdaFactor 换成 Adam 解决了报错的问题

再次得到结果：
但是得到新问题就是：模型训练不起来，准确率不会变化
还请苏神指点，谢谢！！！

**Comment by Lufffya** [comment]
再次尝试：
去查阅了这个问题：tf.function-decorated function tried to create variables on non-first call
发现了一篇文章 https://blog.csdn.net/dou3516/article/details/118097216
发现按照这个方法修改不会报错，可以训练

又去查阅了tfkeras.Model.run_eagerly 属性含义，发现可能是在tf.functions 下面使用 tf. variable 会产生问题

于是发现了这个函数
![image](https://user-images.githubusercontent.com/49063302/148070038-bbb0c8c1-7919-47cb-b56b-968a2837708d.png)

于是当我尝试直接跳过这个函数
将 _decayed_lr 函数下的代码从 return self.learning_rate 改成  return self._learning_rate 

![image](https://user-images.githubusercontent.com/49063302/148070080-68aea6fb-d4aa-40a3-abe4-75da8af4e071.png)

发现可以训练，得到以下结果：
![image](https://user-images.githubusercontent.com/49063302/148070325-6a4aa158-5cf4-4322-8f40-78a9118a0f53.png)

发现 accuracy 可能有些不正常

**Comment by bojone** [comment]
感觉反馈。另外可以考虑disable eager试试
